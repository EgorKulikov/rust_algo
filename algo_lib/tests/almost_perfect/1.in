6
65
650
