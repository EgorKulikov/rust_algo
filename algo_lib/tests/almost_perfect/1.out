6 perfect
65 not perfect
650 almost perfect
